pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: collection
-version: 1.17.2
+version: 1.17.3
```

Comment:
```suggestion
version: 1.18.0-wip
```

Reply:
I can commit the suggestion, but please confirm if you really want to jump to `1.18.0` for this merge.

Best regards.

Reply:
Yes, for new APIs we rev the second number in the semver version.